player/lua/stats.lua

```diff
@@ -1168,7 +1168,11 @@ local function finalize_page(header, content, apply_scroll)
         pages[curr_page].offset = from
     end
     local output = table.concat(header) .. table.concat(content, "", from, to)
-    if not o.use_ass and term_width > 0 then
+    if not o.use_ass and o.term_width_limit == nil and o.bindlist == "no" then
+        local clip = mp.get_property("term-clip-cc")
+        local t = split(output, "\n", true)
+        output = clip .. table.concat(t, "\n" .. clip)
+    elseif not o.use_ass and term_width > 0 then
```

Comment:
Do we need all this `term_width`, `term_ellipsis_array`, `term_width_limit` and so on in stats.lua?

Reply:
Yeah it's still used by `--idle --script-opts=stats-bindlist=yes` as `io.write` bypasses mpv code and `print` / `mp.msg` doesn't work with it because it quits immediately.

Reply:
> as io.write bypasses mpv code
Why we do that? I guess to strip prefixes? Sounds hacky. Do we add `${term-no-prefix-cc}`? Kidding, but only a little, I would prefer scripts to not be too "smart" on its own.

Reply:
2335ee5514 talks about the prefix but `mp.msg.info` doesn't work at all for me there, presumably because it quits right afterwards.

Reply:
So, hacks on hacks... Let me take a look on my end.

Reply:
@guidocella: It is because of `mp.command("no-osd set really-quiet yes")`

Reply:
@guidocella: After recent merge it should be now possible to remove all "width" handling from stats.lua and keep only number of lines for scrolling purposes.

Reply:
Done but this breaks piping to less for some reason. Only the ... remain.

Reply:
Should be fixed by #15117 not sure where we lost this check.

Could you also clean bindlist code at the bottom?